cmd/testbucket/partition.go

```
@@ -0,0 +1,255 @@
+		if hi < lo {
+			hi, lo = lo, hi
+		}
+		tuples = append(tuples[:hi], tuples[hi+1:]...)
+		tuples = append(tuples[:lo], tuples[lo+1:]...)
+		tuples = append(tuples, merged)
+	}
+
+	final := tuples[0]
+	// Recompute true loads: the running `load` values are normalised
+	// differences, not sums.
+	real := make([]kkPart, k)
```

golangci-lint (2.12.2)

[error] variable real has same name as predeclared identifier

(predeclared)